### PR TITLE
docs(guide): run ML on another machine

### PR DESCRIPTION
**Pull request by fabiannagel**
Based on [this discussion](https://github.com/immich-app/immich/discussions/2852#discussioncomment-6223779) from the 1.62.0 release.

### CODE REVIEW

**Comment by bo0tzz** [comment]
Awesome, thank you! To avoid filling the install guide too much, I think it might be better to make it a separate page under the 'Guides' section.

**Comment by fyfrey** [comment]
> Awesome, thank you! To avoid filling the install guide too much, I think it might be better to make it a separate page under the 'Guides' section.

Agreed! It would also be good to discuss in the guide what happens if the other machine is not available (e.g. your laptop is powered off)

File: docs/docs/install/docker-compose.md [hunk 1]
```diff
@@ -154,6 +154,40 @@ IMMICH_MACHINE_LEARNING_URL=http://immich-machine-learning:3003
 - Consider changing `DB_PASSWORD` to something randomly generated
 - Consider changing `TYPESENSE_API_KEY` to something randomly generated
 
+<details>
+<summary>Hosting the machine-learning service on a different system</summary>
+
+To alleviate [performance issues on low-memory systems](/docs/FAQ.md#why-is-immich-slow-on-low-memory-systems-like-the-raspberry-pi) like the Raspberry Pi, you may also host Immich's machine-learning container on a more powerful system (e.g. your laptop or desktop computer):
+
+- Set `IMMICH_MACHINE_LEARNING_URL` to point to the designated ML system, e.g. `http://workstation:3003`
+- Copy your `.env` file and the following `docker-compose.yml` to the ML system:
```
Comment (bo0tzz) [inline docs/docs/install/docker-compose.md:163]: The .env file isn't actually necessary at all for the separate ML container.

**Comment by alextran1502** [comment]
Hello, can you help move the info to the suggested section?

**Comment by alextran1502** [comment]
Hello, any update on this PR?

**Comment by fabiannagel** [comment]
Sorry for the delay guys, I was quite busy the last weeks. Moved to the "Guides" section, hope the title is not too long. Also removed the `.env` file reference. Also added a comment about potential issues due to version mismatches.

**Review by bo0tzz — APPROVED** [review]

**Comment by jrasm91** [comment]
Run npm run format:fix

**Review by jrasm91 — APPROVED** [review]
Looks good. Thank you.

If the container is offline, the app will still try to connect and there will just be a bunch of errors in the server logs I think. We can improve this behavior in the future.

**Pull request merged**